frontend/src/components/PackageGeneration/Editor.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Editor from 'react-simple-code-editor';
+import { highlight, languages } from 'prismjs/components/prism-core';
+import 'prismjs/components/prism-clike';
+import 'prismjs/components/prism-javascript';
+import './Editor.scss'
+
+const code = `a {
+    margin:10px;
+}`;
+
+class editor extends React.Component {
+
+   state = { code };
+   render() {return(
+       <div className="Container">
+        <Editor className="Editor"
+        value={this.state.code}
+        onValueChange={code => this.setState({ code })}
+        highlight={code => highlight(code, languages.js)}
+        padding={10}
+        style={{
```

Comment:
Eventually you might want to pull a lot of the css into a common file and inherit from that

Reply:
I guess this is added as a property to it, I am not too sure if I can pull it out :(